test/e2e/validation_test.go

```diff
@@ -46,6 +46,39 @@ type winService struct {
 	description string
 }
 
+// testKubeletPriority tests if kubelet priority class is set to "AboveNormal"
+func testKubeletPriority(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+	requiredPriorityClass := "AboveNormal"
+
+	t.Run("nodes", func(t *testing.T) {
+		if len(gc.allNodes()) == 0 {
+			t.Skip("0 expected nodes")
+		}
+		for _, node := range gc.allNodes() {
+			out, err := tc.getKubeletPriorityClass(&node)
+			require.NoError(t, err, "error getting kubelet priority class")
+			require.Containsf(t, out, requiredPriorityClass, "node %s missing required kubelet priority class",
```

Comment:
This should be an assert. require will prevent other tests from running after, if something were to fail